.travis.yml

```
@@ -40,8 +40,12 @@ matrix:
       env: TOXENV=py38
       stage: test
 
+    - python: 3.9-dev
```

Since this PR was opened, 3.9 has been released. So we can use that now.

```suggestion
    - python: 3.9
```